src/assets/components/CommentSection.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../../api";
+import Card from "react-bootstrap/Card";
+import Container from "react-bootstrap/Container";
+import Spinner from "react-bootstrap/Spinner";
+import Alert from "react-bootstrap/Alert";
+
+const CommentSection = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  useEffect(() => {
+    if (article_id) {
```

Comment:
I think you would always have an article_id so not sure if this needs to be in an if? Happy to be proven wrong though!